lms/templates/dashboard.html

```diff
@@ -155,27 +155,29 @@ <h2 class="header-courses">${_("My Courses")}</h2>
         <div id="dashboard-search-results" class="search-results dashboard-search-results"></div>
       % endif
 
-      <div class="profile-sidebar" id="profile-sidebar" role="region" aria-label="Account Status Info">
-        <header class="profile">
-          <h2 class="account-status-title sr">${_("Account Status Info")}: </h2>
-        </header>
-        <div class="user-info">
-          <ul>
-
-            % if len(order_history_list):
-            <li class="order-history">
-              <span class="title">${_("Order History")}</span>
-              % for order_history_item in order_history_list:
-                <span><a href="${order_history_item['receipt_url']}" target="_blank" class="edit-name">${order_history_item['order_date']}</a></span>
-              % endfor
-            </li>
-            % endif
-
-            <%include file="${static.get_template_path('dashboard/_dashboard_status_verification.html')}" />
-
-          </ul>
-            </div>
+      % if len(order_history_list) or verification_status in ['approved', 'must_reverify', 'pending', 'expired']:
```

Comment:
@ahsan-ul-haq Can you please take the condition `verification_status in ['approved', 'must_reverify', 'pending', 'expired']` into it's own function? I feel like the condition duplicated like this is prone to dirtiness down the road